client/mapctrl_common.cpp

```diff
@@ -374,41 +374,3 @@ void update_line(int canvas_x, int canvas_y)
     break;
   };
 }
-
-/**
-   We sort according to the following logic:
-
-   - Transported units should immediately follow their transporter (note that
-     transporting may be recursive).
-   - Otherwise we sort by ID (which is what the list is originally sorted
- by).
- */
-static int unit_list_compare(const void *a, const void *b)
```

Comment:
Hmm, I remember something breaking when I removed this sometime ago :smile: But, somehow it works ontop of everything else you did.